python/adxl345_helper.py

```
@@ -0,0 +1,16 @@
+import time
+import board
+import busio
+import adafruit_adxl34x
+
+class ADXL345Sensor:
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)